dev.py

```diff
@@ -352,8 +358,9 @@ def format_code():
 parser.add_argument(
     "--install-at",
     type=str,
+    action='append',  # This allows multiple --install-at arguments
     default=None,
-    help="If path is specified, then builded addon will be also copied to that location.",
+    help="Specify one or more paths where the addon should be installed. Can be used multiple times.",
```

Comment:
Actually you specify just one path. You cannot specify multiple paths after one flag. For that action=extend would be needed.

Valid that one path can be specified multiple times. ---> `Specify path where the add-on should be installed. Flag can be used multiple times.`

Reply:
done